helm/values-prod.yaml

```diff
@@ -73,6 +73,15 @@ microservice-chart:
                 operator: In
                 values:
                   - user
+    podAntiAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+        - weight: 100
+          podAffinityTerm:
+            labelSelector:
+              matchLabels:
+                app.kubernetes.io/instance: nodo
```

Comment:
`node-technicalsupport` instead of `nodo`

Reply:
Done